integration-tests/tests/grub_test.go

```diff
@@ -0,0 +1,45 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+// +build !excludeintegration
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package tests
+
+import (
+	"fmt"
+
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/cli"
+	"github.com/ubuntu-core/snappy/integration-tests/testutils/common"
+
+	"gopkg.in/check.v1"
+)
+
+var _ = check.Suite(&grubSuite{})
+
+type grubSuite struct {
+	common.SnappySuite
+}
+
+func (s *grubSuite) TestGrubBootDirMustNotContainKernelFiles(c *check.C) {
+	for _, targetFile := range []string{"vmlinuz", "initrd.img"} {
+		output, err := cli.ExecCommandErr("find", "/boot/grub", "-name", fmt.Sprintf(`"%s"`, targetFile))
```

Comment:
we have partition.BootDir(), instead of /boot/grub.
That makes me note that we need to rename partition to bootloader.